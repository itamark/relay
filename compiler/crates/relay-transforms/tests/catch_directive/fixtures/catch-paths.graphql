query EnabledFooQuery {
  viewer @catch {
    actor @catch {
      address @catch {
        country @catch
      }
    }
  }
}
