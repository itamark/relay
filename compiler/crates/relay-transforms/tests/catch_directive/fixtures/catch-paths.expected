==================================== INPUT ====================================
query EnabledFooQuery {
  viewer @catch {
    actor @catch {
      address @catch {
        country @catch
      }
    }
  }
}
==================================== OUTPUT ===================================
query EnabledFooQuery {
  viewer @catch @__CatchMetadataDirective
  # CatchMetadataDirective {
  #     to: Result,
  #     path: "viewer",
  # }
   {
    actor @catch @__CatchMetadataDirective
    # CatchMetadataDirective {
    #     to: Result,
    #     path: "viewer.actor",
    # }
     {
      address @catch @__CatchMetadataDirective
      # CatchMetadataDirective {
      #     to: Result,
      #     path: "viewer.actor.address",
      # }
       {
        country @catch @__CatchMetadataDirective
        # CatchMetadataDirective {
        #     to: Result,
        #     path: "viewer.actor.address.country",
        # }
        
      }
    }
  }
}
